QuickBrick/plugins/TViOSLogin/src/LoginScreen/Utils/validation.js

```diff
@@ -1,14 +1,16 @@
 import * as Yup from 'yup';
 
+const message = 'Username or password are not correct.';
+
 const validationSchema = Yup.object().shape({
   username: Yup.string()
     .label('Username')
-    .required('User or password are not correct')
-    .min(4, 'Username must have at least 4 characters '),
+    .required(message)
+    .min(4, message),
   password: Yup.string()
     .label('Password')
-    .required('User or password are not correct')
-    .min(6, 'Password must have at least 6 characters ')
+    .required(message)
+    .min(4, message)
```

Comment:
if we remove 'Password must have at least 6 characters', how the user know why is not valid ?

Reply:
It is login screen (not signup screen), so it was decided to remove that hint for login screen for security reasons. But for sign up screen, you are totally right, it will be added to help user create secure password.